api/json-schema/schema.json

```diff
@@ -18767,13 +18767,13 @@
     },
     "io.numaproj.numaflow.v1alpha1.PipelineLimits": {
       "properties": {
-        "bufferMaxLength": {
-          "description": "BufferMaxLength is used to define the max length of a buffer Only applies to UDF and Source vertices as only they do buffer write. It can be overridden by the settings in vertex limits.",
+        "partitionMaxLength": {
```

Comment:
Will `isbPartitionMaxLength` be more clear that this is a ISB partition? Same for other properties below.